src/App.jsx

```diff
@@ -0,0 +1,94 @@
+import "./App.css";
+import contactsData from"./contacts.json"
+import { useState } from "react";
+
+
+function App() {
+  const [contacts,setContacts] = useState(contactsData.slice(0,5))
+  console.log(contacts)
+
+  function randomContacts(){
+    let copiedArray = [...contacts]
```

Comment:
We need a guard safe in order to not add more contacts that we have could be something like this 

```suggestion
   if (contactsData.length === contacts.length) {
      console.log("No more contacts can be add")
      return
    }
    let copiedArray = [...contacts]
```